1907-01-21.xml

```diff
@@ -0,0 +1,1506 @@
+<?xml-stylesheet type="text/xsl" href="dig-eg-gaz.xsl"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title level="m" type="main">Digital Egyptian Gazette</title>
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor>Ryan Jones</editor>
+                <principal>Will Hanley</principal>
+            </titleStmt>
+            <editionStmt>
+                <edition>
+                    <date when="2018-01-17">January 17, 2018</date>
+                </edition>
+            </editionStmt>
+            <publicationStmt>
+                <publisher>FSU University Libraries</publisher>
+                <pubPlace>Tallahassee, FL</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title>The Egyptian Gazette</title>
+                    <date when="1907-01-21"><name>Monday</name>, January 21, 1907</date>
+                    <extent><measure unit="pages" quantity="8">8</measure> pages</extent>
+                </bibl>
+            </sourceDesc>
+        </fileDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    <head type="sub">No. <measure quantity="7666">7,666</measure>] ALEXANDRIA,
+                        MONDAY, JANUARY 21, 1907. [EIGHT PAGES P.T. 1. </head>
+                </div>
+                <div type="advert" xml:id="deg-ad-etc01">
+                    <head>The Eastern Telegraph Company, Limited.</head>
+                    <p>This Company's system of submarine telegraph <lb/>cables is the most direct
+                        and quickest means of <lb/>communication from Egypt to Europe, North and
+                        <lb/>South America, East, South and West Africa, <lb/>India, Australia, New
+                        Zealand, China and Japan.</p>
+                    <p>To secure quick transmission, telegrams should <lb/>be marked <hi>Via
+                            Eastern.</hi></p>
+                    <p>For latest average time to London, see daily <lb/>bulletin in this paper.</p>
+                    <p>STATIONS IN EGYPT: Alexandria, Cairo, <lb/>Suez, Port-Tewfik, Port-Saïd,
+                        Suakin. Head <lb/>Office. London.</p>
+                </div>
+                    <div type="advert" xml:id="deg-ad-sfp01">
+                        <head>Throw Steel Nibs Away!</head>
+                        <p>They waste time, are costly to use, wear out, scratch, etc.</p>
+                        <p>Try a "Swan" Fountain Pen.</p>
+                        <p>Prices from 10/6</p>
+                        <p>Sold by all Stationers and Importers in Cairo and Alexandria.</p>
+                        <p>Catalogue free.</p>
+                        <p>Mable, Todd &amp; Bard, High Holborn, London.</p>
```

Comment:
This ad is missing a </div> tag to close it. There <div type="section"> is also missing. Check the template to see if you can fix this up.